### DISCUSSION TITLE
'Fix through the Netfabb' function

### DISCUSSION THREAD

**Discussion by axg20202** [supermerill/SuperSlicer#2949] [General]
This is a very useful feature of PS.  Will it ever be brought across to SS?  It seems everyone is keen to have new PS features merged into SS without delay, but this is a really old feature.

**Comment by supermerill** [comment]
I just tried, and it works.
It's only available on windows 10. You have to click on the warning icon in the object list.

Is there something else i missed?

**Reply by axg20202** [reply]
I'm on Win 11. Clicking the warning icon does nothing apart from bringing up the mouse-over box stating that the item was autorepaired and the number of errors that were fixed. The thing is, the autorepair does not fix all the issues that are fixed by Netfabb. There doesn't appear to be any way to manually force a repair.

**Reply by neophyl** [reply]
On windows 10 you can right click the Object either on the Plater or in the Object tree listing and the Repair through netfabb option is available in the list.

**Reply by axg20202** [reply]
Really? That option is available on my install of PrusaSlicer, but not on SuperSlicer.

**Reply by axg20202** [reply]
Well, don't I feel silly - I just tried a slightly more recent version of SS and the function is there. Sorry to disturb!
